assets/js/components/KeyMetrics/key-metrics-widgets.js

```diff
@@ -381,6 +382,19 @@ const KEY_METRICS_WIDGETS = {
 		],
 		displayInList: shouldDisplayWidgetWithConversionEvent,
 	},
+	[ KM_ANALYTICS_TOP_CITIES_DRIVING_PURCHASES ]: {
+		title: __( 'Top cities driving purchases', 'google-site-kit' ),
+		description: __(
+			'Cities driving the most purchases',
+			'google-site-kit'
+		),
+		infoTooltip: __(
```

Comment:
The text was provided and approved by Andrey, for both description and tooltip. I won't be able to change that as part of this issue, as any new suggestion would need to go through his approval. Since all ACR tiles will have same description and info tooltip, I will see to open a follow-up issue if needed to suggest different text for tooltips, will check with his preference first.

Reply:
Thanks, @zutigrm. My bad, i mixed up descriptions and titles in my head. All good.